### ISSUE TITLE
QuickInput error aligns weirdly

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#145672]
Testing #145626

Notice how the quick input validation message aligns weirdly with the rest of quick input:
1. It overwrites the placeholder text to "Accept on enter..."
2. It goes below the quick input

It feels like it was just stuck on top of the existing quick input UI. I would love it feel a bit more native in the quick input.

![Screenshot 2022-03-22 at 12 18 57](https://user-images.githubusercontent.com/1926584/159470393-66566f5e-37ea-4cc8-ab4f-f3c8a72e79b2.png)

**Comment by TylerLeonhardt** [comment]
@isidorn so before, it would cause the quickpick to grow... and someone opened an issue saying that looked weird. I think it was @bpasero.

Even though it goes below the quick input, I think the layering looks fine because of the thick border.

> I would love it feel a bit more native in the quick input.

what would this look like in your opinion?

**Comment by bpasero** [comment]
Yeah I am not a big fan of how it was and how it is now but find the current behaviour better :)

**Comment by isidorn** [comment]
It would feel more native as if the dimensions were aligned with the quick pick dimensions, and it was not going outside of it.

I think it is fine as it is... So feel free to leave it. Or if you would like to polish then maybe we can ping the design team for help.

**Comment by TylerLeonhardt** [comment]
@misolori  do you have thoughts about this?

**Comment by misolori** [comment]
We could alternately use the quick pick entries to highlight the error instead? Though that would be a new pattern for errors:

<img width="310" alt="CleanShot 2022-03-23 at 11 55 41@2x" src="https://user-images.githubusercontent.com/35271042/159775053-0c66abc2-f212-4128-81ba-88e27341f203.png">

Similarly used:

<img width="241" alt="CleanShot 2022-03-23 at 11 56 00@2x" src="https://user-images.githubusercontent.com/35271042/159775108-82f8c891-5b5a-46aa-864f-68fb3aa90431.png">

**Comment by isidorn** [comment]
@misolori this looks great but more aligned with the QuickPick UI. But I wonder if the message is multi line?

**Comment by TylerLeonhardt** [comment]
Yeah today the "No matching results" part can't be multi-line:

![Image](https://user-images.githubusercontent.com/2644648/206007591-e2db97f0-0598-4ceb-bdb4-d1a2900a0f3c.png)

However, I think there's more to discuss here... one of the problems I have with the current experience is that the error _covers_ the prompt. So if you forget the prompt, you have to mutate the input to stop showing the error to see what you actually need to input.

I think we need to step back and put the error somewhere less disruptive. Asking questions out loud:

* Does the error maybe push the prompt down? might make the prompt too jumpy
* Does the error go underneath the prompt? might make the error too disconnected
* Does the error go to the right? quick pick isn't really wide enough for that
* Does the error go above? this might cause the whole input/prompt to get pushed down so again too jumpy
* Do we go the squiggles route similar to the editor? might not be in the users face enough

**Comment by TylerLeonhardt** [comment]
Another one for @daviddossett to think about maybe? Also seems applicable to other quick pick/command center stuff.
